src/app/core/language/language-select/language-select.component.ts

```diff
@@ -8,6 +8,8 @@ import { MatMenuModule } from "@angular/material/menu";
 import { NgForOf } from "@angular/common";
 import { LOCALE_ENUM_ID } from "../languages";
 import { ConfigurableEnumDirective } from "../../basic-datatypes/configurable-enum/configurable-enum-directive/configurable-enum.directive";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
```

Comment:
```suggestion
```

Looks like IconProp is not actually used anymore

Reply:
Ok I will remove it